src/styles/index.scss

```diff
@@ -1,3 +1,220 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+
+  --link-color: #00acdc;
+  --header-height: 60px;
+  --column-count: 1;
```

Comment:
The CSS variables '--link-color', '--header-height', and '--column-count' are correctly defined for repeated sizes and special colors, as required by the checklist.